### DISCUSSION TITLE
Hostname for <Image> fetches from hotlink-protected CDN?

### DISCUSSION THREAD

**Discussion by HDv2b** [vercel/community#173] [Help] [answered]
I'm working on a video sharing site with content hosted on a CDN (_Bunny CDN_ to be precise). I'm using <Image> to show the thumbnails in a video gallery.

In Bunny CDN, I've enabled the block for direct file URL access, which opting in to whitelist allowed hostnames:

> Block Direct URL File Access
> If enabled, direct access to your files without a referer header will be blocked.

But since next.js is fetching the image in the backend, I'm not sure either what hostname to set in the cdn config, or how to set what referal header for nextjs to use?

**Comment by Pieparker** [comment, marked as answer]
I would have thought that the `hostname` would have been the domain(s) used on your site. It may be worth speaking to BunnyCDN for clarification here, if this isn't the case!
